azure-devops.rest

```diff
@@ -1,72 +1,73 @@
-@apiVersionParam = api-version=5.0
-@apiVersionPreviewParam = api-version=5.1-preview.2
+{{
+    apiVersionParam = "api-version=7.1"
+    apiVersionPreviewParam = "api-version=7.1-preview.3"
+}}
```

Comment:
What is this syntax? I have been using VSCode Rest Client extension here but this looks like something else. I can't see this syntax in the docs.
I'm not against changing the tool if it's better but I would like to know what is it.